internal/execext/exec.go

```diff
@@ -103,6 +103,9 @@ func IsExitError(err error) bool {
 func Expand(s string) (string, error) {
 	s = filepath.ToSlash(s)
 	s = strings.ReplaceAll(s, " ", `\ `)
+	s = strings.ReplaceAll(s, "&", `\&`)
+	s = strings.ReplaceAll(s, "(", `\(`)
+	s = strings.ReplaceAll(s, ")", `\)`)
```

Comment:
Should this not do something along the lines of "shell quote the string"? Instead of escaping individual characters one at a time?

Reply:
yeah - something like https://pkg.go.dev/github.com/frioux/shellquote@v0.0.2 seems more complete